pallets/eth-bridge/src/lib.rs

```diff
@@ -651,6 +668,81 @@ pub mod pallet {
 
         Ok(())
     }
+    fn has_author_casted_event_vote<T: Config>(author: &T::AccountId) -> bool {
```

Comment:
again, no vote identifier. Does any vote count? We are ignoring the partition differences themselves, so any vote anywhere will make this return true. What is the meaning of this function, overall?